venues/MIDL.io/2019/Conference/python/recruit_area_chairs.py

```diff
@@ -0,0 +1,82 @@
+import sys, os
+import argparse
+import openreview
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+builder = openreview.conference.ConferenceBuilder(client)
+builder.set_conference_id('MIDL.io/2019/Conference')
+builder.set_conference_name('Medical Imaging with Deep Learning')
+builder.set_homepage_header({
+'title': 'Medical Imaging with Deep Learning',
+'subtitle': 'MIDL 2019 Conference',
+'deadline': 'Submission Deadline: 13th of December, 2018',
+'date': '8-10 July 2019',
+'website': 'http://2019.midl.io',
+'location': 'London',
+'instructions': 'Full papers contain well-validated applications or methodological developments of deep learning algorithms in medical imaging. There is no strict limit on paper length. However, we strongly recommend keeping full papers at 8 pages (excluding references and acknowledgements). An appendix section can be added if needed with additional details but must be compiled into a single pdf. The appropriateness of using pages over the recommended page length will be judged by reviewers. All accepted papers will be presented as posters with a selection of these papers will also be invited for oral presentation.'
+})
+builder.set_conference_type(openreview.builder.SingleBlindConferenceType)
```

Comment:
Do you need to set the homepage_header to be able to use the conference_builder to send the recruiting emails?

Reply:
Yes, the invitation response web page it show this header.